### PR TITLE
[Base] Support multiple outputs for split operation

### PR DESCRIPTION
**Pull request by fujunwei**

### CODE REVIEW

File: src/webnn_native/Operand.cpp [hunk 1]
```diff
@@ -21,8 +21,14 @@
...
 
-    OperandBase::OperandBase(GraphBuilderBase* graphBuilder, std::vector<Ref<OperandBase>> inputs)
-        : ObjectBase(graphBuilder->GetContext()), mInputs(std::move(inputs)) {
+    OperandBase::OperandBase(GraphBuilderBase* graphBuilder, OperatorBase* operatorBase)
+        : ObjectBase(graphBuilder->GetContext()), mOperator(operatorBase) {
+        if (mOperator->Inputs().size() >= 1) {
+            auto primaryInput = mOperator->Inputs()[0];
+            // The type and rank is the same as input[0] by default.
+            mType = primaryInput->Type();
+            mRank = primaryInput->Rank();
```
Comment (huningxin) [inline src/webnn_native/Operand.cpp:30]: Would the default type and rank be necessary?

File: src/webnn_native/Operand.h [hunk 2]
```diff
@@ -21,36 +21,41 @@
...
 #include "webnn_native/ObjectBase.h"
+#include "webnn_native/Operator.h"
 #include "webnn_native/webnn_platform.h"
 
 namespace webnn_native {
 
     class OperandBase : public ObjectBase {
       public:
-        explicit OperandBase(GraphBuilderBase* GraphBuilder, std::vector<Ref<OperandBase>> = {});
+        OperandBase(GraphBuilderBase*, OperatorBase*);
```
Comment (huningxin) [inline src/webnn_native/Operand.h:31]: keep explicit?

**Review by huningxin — APPROVED** [review]
Thanks @fujunwei , this PR makes the infrastructure more flexible for future development. LGTM with two opens. Please take a look.

File: src/webnn_native/Operand.cpp (hunk 1 above)
Comment (fujunwei) [reply src/webnn_native/Operand.cpp:30]: Type can be float32 by default, is it 0 for the rank?

File: src/webnn_native/Operand.h (hunk 2 above)
Comment (fujunwei) [reply src/webnn_native/Operand.h:31]: [The explicit](https://google.github.io/styleguide/cppguide.html)  is used for single-argument constructors and preventing implicit conversions (int to double), `GraphBuilderBase* `and `OperatorBase*` is not implicit conversions, so explicit  is not necessary.

Comment (huningxin) [reply src/webnn_native/Operand.h:31]: cool, thanks for the explanation.

**Comment by fujunwei** [comment]
Let's merge it, thanks.

**Pull request merged**